googleapis_auth/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.5.1
 
 - `auth_browser.dart`: handle pop-up closed errors correctly. 
+- Added support for custom post-auth html page
```

Comment:
Add a 1.6.0 section above and add this. Update the pubspec accordingly!

Reply:
> Add a 1.6.0 section above and add this. Update the pubspec accordingly!

Done